cinema/serializers.py

```diff
@@ -59,6 +60,7 @@ class MovieSessionListSerializer(MovieSessionSerializer):
     cinema_hall_capacity = serializers.IntegerField(
         source="cinema_hall.capacity", read_only=True
     )
+    tickets_available = serializers.IntegerField(read_only=True, source="cinema_hall.ticavailable")
```

Comment:
There is a typo in the source for `tickets_available`. It should be `cinema_hall.tickets_available` instead of `cinema_hall.ticavailable` to match the property defined in the `CinemaHall` model.